flink-contrib/docker-flink/Dockerfile

```diff
@@ -36,22 +31,24 @@ ENV PATH $PATH:$FLINK_HOME/bin
 EXPOSE 8081
 EXPOSE 6123
 
+# flink-dist can point to a directory or a tarball on the local system
+ARG flink_dist=NOT_SET
+
 # Install build dependencies and flink
+ADD $flink_dist $FLINK_INSTALL_PATH
 RUN set -x && \
-  mkdir -p $FLINK_INSTALL_PATH && \
-  apk --update add --virtual build-dependencies curl && \
-  curl -s $(curl -s https://www.apache.org/dyn/closer.cgi\?preferred\=true)flink/flink-${FLINK_VERSION}/flink-${FLINK_VERSION}-bin-hadoop${HADOOP_VERSION}-scala_${SCALA_VERSION}.tgz | \
```

Comment:
Which existing image are you referring to that we need to keep updated? I'm not aware of a Flink project- or data Artisans-maintained image on Docker Hub.

I'm almost done with the basic functionality for the first Alpine image (and a script to template its Dockerfile) but there will be plenty of room for improvements. Once the new repo is live I'll ping you and you can take a look.

I don't know if I need to be the "official maintainer", but the repo will live under the [data Artisans GitHub org](https://github.com/dataArtisans), so if not me someone at dA will likely handle contributions to that repo.